charts/prefect-orion/values.yaml

```diff
@@ -32,10 +32,12 @@ orion:
     # -- enable orion image debug mode
     debug: false
 
+  # -- PREFECT_ORION_UI_API_URL; should be publically accesible API url; default to PREFECT_API_URL for running API locally
```

Comment:
I'm not sure "default to PREFECT_API_URL for running API locally" applies here since the API is running in a cluster by definition here?

Reply:
May also want to note that if a value is not provided the UI will not work.

Reply:
Yeah but they won't have set `PREFECT_API_URL` and they shouldn't here so it just adds confusion

Reply:
`PREFECT_API_URL` is null by default so that it defaults to that value isn't really helpful for them here. They shouldn't use `PREFECT_API_URL` API in this context so I think it just adds confusion.